fbgemm_gpu/codegen/embedding_backward_split_template.cu

```diff
@@ -930,11 +930,11 @@ split_embedding{{ "_nobag" if nobag else "" }}_backward_codegen_{{ optimizer }}_
                         ->philox_cuda_state(4);
             }
             {% endif %}
-            {% for kMaxVecsPerThread in range(1, max_embedding_dim // 128 + 1) %}
```

Comment:
Is `128` only hardcoded in this kernel ? 

Is `ITEMS_PER_CTA` too generic of a name, since other kernels might also have similar structure and 'items/cta` ?